### PR TITLE
ci: Fix failing Flutter dry-run job due to missing changelog version entry

### PR DESCRIPTION
**Pull request by Nidal-Bakir**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).

### Issue Description

Closes: #856

### Approach

setup Flutter rather than only Dart

### TODOs before merging
N/A

- [ ] Add tests
- [ ] Add changes to documentation (guides, repository pages, in-code descriptions)
- [ ] A changelog entry

### CODE REVIEW

**Comment by Nidal-Bakir** [comment]
@mtrezza  
The CI issue fixed 

---
> Parse-SDK-Flutter/Parse-SDK-Flutter/packages/flutter/CHANGELOG.md doesn't mention current version (4.0.0).
>   Consider updating it with notes on this version prior to publication.

its now complaining about not mentioning the new 4.0.0 version in the CHANGELOG like the other release versions 

* the dry-run expect something like this :

  > ## [3.1.4](https://github.com/parse-community/Parse-SDK-Flutter/compare/flutter-3.1.3...flutter-3.1.4) (2023-03-01)
  > 
  > ### Bug Fixes
  > 
  > * Parse SDK internal database file `parse.db` is accessible for app user on iOS and may be accidentally deleted ([#826](https://github.com/parse-community/Parse-SDK-Flutter/pull/826))

* What is present in the CHANGELOG : 

  > ## Unreleased (master branch)
  > 
  > ### BREAKING CHANGES
  > 
  > * The source file name of the Flutter package has changed; to import the Flutter package use `parse_server_sdk_flutter.dart` instead of `parse_server_sdk.dart` ([#846](https://github.com/parse-community/Parse-SDK-Flutter/pull/846))
  > 
  > ### Features
  > 
  > * Rename Flutter package source file to `parse_server_sdk_flutter.dart` ([#846](https://github.com/parse-community/Parse-SDK-Flutter/pull/846))

**Comment by mtrezza** [comment]
Is it just looking for the string `4.0.0`? Could you try changing the changelog line to `Unreleased (4.0.0)`

**Comment by Nidal-Bakir** [comment]
Yes, that will make it happy. should I open a new PR to fix this?

File: packages/flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Unreleased (master branch)
+## Unreleased (4.0.0)
```
Comment (mtrezza) [inline packages/flutter/CHANGELOG.md:1]:
```suggestion
## 4.0.0 (unreleased)
```

**Comment by mtrezza** [comment]
I think this is good now

**Pull request merged**